lib/grape/validations/params_scope.rb

```diff
@@ -232,16 +232,25 @@ def validates(attrs, validations)
         default = validations[:default]
         doc_attrs[:default] = default if validations.key?(:default)
 
-        values = options_key?(:values, :value, validations) ? validations[:values][:value] : validations[:values]
+        values = validations[:values]
+        if values.is_a? Hash
+          excepts = values[:except]
+          values = values[:value]
+          values = nil if values.is_a? Proc
```

Comment:
I'm looking for a little different behavior that's tricky with a one-liner.  I really don't ever want `values` to be a Hash.  But that's what the one-liner above will do if the Hash doesn't have a `value` attribute.  Fair point about reassigning variables though.  I'll push another commit that clarifies it a bit (I hope).